### PR TITLE
Updating Lores with Profile Stats

### PR DESCRIPTION
**Pull request by L1ghtRay**
I did a pr for the boots to see if the number of fairy souls collected should be randomized.
Since it was commited I updated it for the other 3 pieces.

### CODE REVIEW

**Comment by L1ghtRay** [comment]
I found another armour set with non randomised personal stats, so there might be more of them

**Comment by L1ghtRay** [comment]
oops

File: items/FAIRY_CHESTPLATE.json [hunk 1]
```diff
@@ -14,11 +11,15 @@
     "",
     "§6Full Set Bonus: Fairy's Outfit",
     "§7Gain §c1❤ §7per Fairy Soul.",
-    "§7You found: §d16 Fairy Souls",
+    "§7You found: §d§k000§d Fairy Souls",
```
Comment (IRONM00N) [inline items/FAIRY_CHESTPLATE.json:14]: does the obfuscated text actually get reset here?

Comment (L1ghtRay) [reply items/FAIRY_CHESTPLATE.json:14]: wdym by reset

Comment (IRONM00N) [reply items/FAIRY_CHESTPLATE.json:14]: is the text after the numbers de-obfuscated?

Comment (L1ghtRay) [reply items/FAIRY_CHESTPLATE.json:14]: yes, it works as intended only the 000's get obfuscated

**Comment by L1ghtRay** [comment]
should i close this pr and make a new one which is proper?
cause this is messy now

**Comment by IRONM00N** [comment]
> should i close this pr and make a new one which is proper?
> cause this is messy now

no

**Comment by L1ghtRay** [comment]
> 
> 
> > should i close this pr and make a new one which is proper?
> > cause this is messy now
> 
> no

ok, i will try to be as clean as possible

**Comment by IRONM00N** [comment]
It doesn't matter if it looks messy

**Review by L1ghtRay — COMMENTED** [review]
bruh how did this come here

File: items/PARTY_HAT_CRAB.json [hunk 2]
```diff
@@ -14,14 +11,15 @@
     "§7on your head for each SkyBlock",
     "§7year you\u0027ve been playing.",
     "",
-    "§7Your bonus: §b+0✎ Intelligence",
```
Comment (jani270) [inline items/PARTY_HAT_CRAB.json:17]: Why did you remove this

**Review by jani270 — CHANGES_REQUESTED** [review]

File: items/PARTY_HAT_CRAB.json (hunk 2 above)
Comment (L1ghtRay) [reply items/PARTY_HAT_CRAB.json:17]: It doesn't exist anymore
I asked Nopo to give me the nbt data of the item when he put it on his head and it didn't show that anymore

Comment (jani270) [reply items/PARTY_HAT_CRAB.json:17]: oh you are right nvm then

**Review by jani270 — APPROVED** [review]

**Review by IRONM00N — APPROVED** [review]
